home/management/commands/create_default_site.py

```diff
@@ -1,16 +1,33 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
-from django.contrib.sites.models import Site
+from django.contrib.sites.models import Site as DjangoSite
 from django.core.management.base import BaseCommand
+from wagtail.core.models import Site as WagtailSite
+
+from home.models import HomePage
 
 User = get_user_model()
 
 
 class Command(BaseCommand):
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--domain',
+            help='The domain on which the website is deployed'
+        )
+
     def handle(self, *args, **options):
-        _, created = Site.objects.update_or_create(pk=settings.SITE_ID, defaults={
-            'domain': 'localhost:8000',
+        domain = options.get('domain') or 'localhost:8000'
```

Comment:
It appears that the DjangoSite domain does not contain a port (at least, if I specify a port, I get an error when posting a comment, see #836, while it works ok if I only specify the URL), so the default should probably be simply `localhost`?